zerver/webhooks/airbrake/doc.md

```diff
@@ -0,0 +1,23 @@
+Get Zulip notifications for your Airbrake bug tracker!
+
+First, create the stream you'd like to use for Airbrake notifications, and
+subscribe all interested parties to this stream. We recommend the
+name `airbrake`.
+
+Next, on your {{ settings_html|safe }}, create an Airbrake bot. Construct the
+URL for the Airbrake bot using the API key and stream name:
+
+`{{ external_api_uri_subdomain }}/v1/external/airbrake?api_key=abcdefgh&stream=airbrake`
```

Comment:
Hmmm... I don't know about the tooling in this area yet, but which templating engine goes through this?